source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveCardRenderer.cs

```diff
@@ -82,9 +83,27 @@ public ResourceDictionary Resources
                     _resources = new ResourceDictionary();
                 }
 
+                var resource = new ResourceDictionary
+                {
+                    Source = new Uri("/AdaptiveCards.Rendering.Wpf;component/Themes/generic.xaml",
```

Comment:
This is causing an exception to be thrown is the tests to fail the first time Resources are accessed.